### PR TITLE
Update Kotlin to 1.1.51

### PR DESCRIPTION
**Pull request by RebelliousUno**
Also update forge for MC 1.12.2

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -2,9 +2,9 @@ mod_version = 1.5.2
 group = net.shadowfacts
 archivesBaseName = Forgelin
 
-mc_version = 1.12
-mcp_mappings = snapshot_20170629
-forge_version = 14.21.1.2387
+mc_version = 1.12.2
```
Comment (faceofcat) [inline gradle.properties]: this should be on 1.12 still... to make sure 1.12, 1.12.1 and 1.12.2 all work well with it.

building against 1.12.2 would probably not be that bad... but only if this version doesn't get enforced as the minimum required version.

Comment (RebelliousUno) [reply gradle.properties]: Sure, I can revert this

**Comment by shadowfacts** [comment]
Forgelin 1.6.0 has been released with Kotlin 1.2.0.

**Comment by RebelliousUno** [comment]
Closing as updated to 1.2

**Comment by RebelliousUno** [comment]
Hah beat me to it :)

**Pull request closed without merging**